hwilib/devices/trezor.py

```diff
@@ -275,6 +275,14 @@ def get_path_transport(
     raise BadArgumentError(f"Could not find device by path: {path}")
 
 
+def _use_external_script_type(client) -> bool:
+    if client.features.model == "1" and client.version > (1, 10, 5):
```

Comment:
This is better and matches more how people think:
```suggestion
    if client.features.model == "1" and client.version >= (1, 11, 1):
```